snapcraft.yaml

```diff
@@ -878,26 +878,33 @@ parts:
       - --enable-virtfs
       - --firmwarepath=/snap/lxd/current/share/qemu/
       - --localstatedir=/var/
+      - --disable-install-blobs # Ubuntu sources don't have the ROM blobs in them.
     build-packages:
       - bison
       - bzip2
       - flex
       - pkg-config
       - libaio-dev
       - libcap-ng-dev
+      - libfdt-dev
       - libglib2.0-dev
       - libnuma-dev
       - libpixman-1-dev
       - librbd-dev
       - libusbredirhost-dev
+      - quilt
     stage-packages:
       - genisoimage
+      - ipxe-qemu # This is needed due to --disable-install-blobs.
```

Comment:
This package has the same version in 22.04 and 24.04 (`1.21.1+git-20220113.fbbdc3926-0ubuntu1`) which points to this upstream commit https://github.com/ipxe/ipxe/commit/fbbdc3926 from Jan 2022. Upstream is pretty active so maybe we'll be missing on some recent bug fixes when moving from the git submodule to the Ubuntu package.

The change in this Ubuntu specific (not from Debian) package was from Christian Ehrhardt so if we need fresher versions, we could always go to him.

Reply:
The same as above:
```
git submodule update --init
make -C roms pxerom
```

resulting file should be `pc-bios/pxe-virtio.rom`

P.S. Have not tested that.

Reply:
@mihalicyn ATM, we only care about `efi-virtio.rom` but I'm assuming that's a moot point.

Reply:
my bad, sure. Then we need `make -C roms efirom` and `pc-bios/efi-virtio.rom` file.